heron/shell/src/python/handlers/downloadhandler.py

```diff
@@ -34,7 +35,14 @@ class DownloadHandler(tornado.web.RequestHandler):
   def get(self, path):
     """ get method """
 
-    logging.debug("request to download: %s", path)
+    handler = logging.StreamHandler()
+    handler.setFormatter(anticrlf.LogFormatter('%(levelname)s:%(name)s:%(message)s'))
+    logger = logging.getLogger(__name__)
```

Comment:
I am wondering if we should move this to a class variable so that no need to set the formatter every time. 

Maybe not a big deal fro downloader though since downloading doesn't happen very often.